app/obfuscators/base64_no_padding.py

```diff
@@ -0,0 +1,30 @@
+from base64 import b64encode
+
+from app.utility.base_obfuscator import BaseObfuscator
+
+
+class Obfuscation(BaseObfuscator):
+
+    @property
+    def supported_platforms(self):
+        return dict(
+            windows=['psh'],
+            darwin=['sh'],
+            linux=['sh']
+        )
+
+    def run(self, link, **kwargs):
+        link.command = link.command.replace('=', '')
+        return super().run(link)
+
+    @staticmethod
```

Comment:
I don't think we need static here, either that or the `self` argument should be removed. Same for the `psh` method below.

Reply:
Done.